website/static/js/fangorn.js

```diff
@@ -607,9 +607,12 @@ function _fangornLazyLoadOnLoad (tree) {
  * @private
  */
 function _fangornOrderFolder(tree) {
-    var sortDirection = this.isSorted[0].desc ? 'desc' : 'asc';
-    tree.sortChildren(this, sortDirection, 'text', 0);
-    this.redraw();
+    // Checking if this column does in fact have sorting
+    if (this.isSorted[0]) {
```

Comment:
It contains the sorted values for each of the possible columns that are sorted and its direction. It can be refactored so that the entire table has current sort state as the column like

```tb.isSorted : { col : 0, direction: 'asc' }```

This makes much more sense but let's talk about whether we want to implement it this way. It's not required for this PR to go forward.